<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NCRDRunnerDialogUI</class>
 <widget class="QDialog" name="NCRDRunnerDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run report</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="gbOutput">
       <property name="title">
        <string>Report output</string>
       </property>
       <layout class="QVBoxLayout" name="layBg">
        <item>
         <widget class="QRadioButton" name="rbPreview">
          <property name="text">
           <string>NCReport preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbQtPreview">
          <property name="text">
           <string>Qt print preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbPrinter">
          <property name="text">
           <string>Printer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbPdf">
          <property name="text">
           <string>PDF file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbSvg">
          <property name="text">
           <string>SVG file(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbImage">
          <property name="text">
           <string>Image file(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbHtml">
          <property name="text">
           <string>HTML file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbExcel">
          <property name="text">
           <string>Excel file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbText">
          <property name="text">
           <string>Text file</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabBatch">
        <attribute name="title">
         <string>Batch mode</string>
        </attribute>
        <layout class="QVBoxLayout" name="layBatch">
         <item>
          <widget class="QCheckBox" name="cbBatchMode">
           <property name="text">
            <string>Run reports in batch mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="batchListWidget">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabParams">
        <attribute name="title">
         <string>Parameters</string>
        </attribute>
        <layout class="QVBoxLayout" name="layParams">
         <item>
          <widget class="QWidget" name="widget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Settings</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="wPageRange"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Page range:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbPreview</tabstop>
  <tabstop>rbQtPreview</tabstop>
  <tabstop>rbPrinter</tabstop>
  <tabstop>rbPdf</tabstop>
  <tabstop>rbSvg</tabstop>
  <tabstop>rbImage</tabstop>
  <tabstop>rbHtml</tabstop>
  <tabstop>rbExcel</tabstop>
  <tabstop>rbText</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NCRDRunnerDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NCRDRunnerDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
